widget/embedded/src/pages/Home.tsx

```diff
@@ -28,15 +28,6 @@ import { getPriceImpact, getPriceImpactLevel } from '../utils/quote';
 import { canComputePriceImpact, getSwapButtonState } from '../utils/swap';
 import { formatBalance, isFetchingBalance } from '../utils/wallets';
 
-const Container = styled('div', {
-  display: 'flex',
-  flexDirection: 'column',
-  overflowY: 'visible',
-  '& .quote__container': {
-    paddingTop: '$2',
-  },
```

Comment:
If we eliminate styles for specific class names, we should also remove the associated class name from the corresponding element.

Reply:
fixed.